identity-apps-core/apps/recovery-portal/src/main/webapp/password-recovery.jsp

```diff
@@ -366,30 +414,17 @@
                         %>
                         <div class="mt-4">
                             <%
-                               if (isEmailNotificationEnabled && isNotificationBasedPasswordRecoveryEnabledByTenant
-                                                    && !isQuestionBasedPasswordRecoveryEnabledByTenant) {
-
+                                String submitButtoni18nText = multipleRecoveryOptionsAvailable? "Submit" :
+                                            ( isEmailRecoveryAvailable? "password.recovery.button.email.link" :
```

Comment:
Reduce tab space to one tab

Reply:
Addressed with https://github.com/wso2/identity-apps/commit/92391d27c88da1fa44eb80d1119749f39fbb258d